src/view/scroll_view.rs

```diff
@@ -58,27 +62,22 @@ where
         id: &mut Id,
         state: &mut Self::State,
         element: &mut Self::Element,
-    ) -> bool {
+    ) -> ChangeFlags {
         cx.with_id(*id, |cx| {
             let child_element = element.child_mut().downcast_mut().unwrap();
-            let changed =
-                self.child
-                    .rebuild(cx, &prev.child, &mut state.0, &mut state.1, child_element);
-            if changed {
-                element.child_mut().request_update();
-            }
-            changed
+            self.child
+                .rebuild(cx, &prev.child, &mut state.0, &mut state.1, child_element)
         })
     }
 
-    fn event(
+    fn message(
         &self,
         id_path: &[Id],
         state: &mut Self::State,
         event: Box<dyn Any>,
```

Comment:
```suggestion
        message: Box<dyn Any>,
```

Reply:
Could you explain the difference between "message" and "event" to me? I'm very surprised that there are two functions with identical signatures that seemingly do a similar thing!

Reply:
I think originally (in the druid [xilem_tokio](https://github.com/linebender/druid/tree/xilem_tokio) branch) it was called event (and was put unmodified into this repo), and was renamed later to message, I'm not entirely sure (Raph probably has a better answer), just a naming thing.

I think it's because `message` kind of implies that it "messages" a general payload, while event doesn't necessarily.